src/wallet/wallet.h

```diff
@@ -745,8 +749,13 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
      * selected by SelectCoins(); Also create the change output, when needed
      * @note passing nChangePosInOut as -1 will result in setting a random position
      */
+<<<<<<< 08b37c5e06bf1698c1d0e5905806382cbfa8cefd
```

Comment:
It also breaks bisections.

In general, the state of the tree should be valid after every commit
(compile and test fine).